actors/miner/src/lib.rs

```diff
@@ -114,6 +116,8 @@ pub enum Method {
     PreCommitSectorBatch = 25,
     ProveCommitAggregate = 26,
     ProveReplicaUpdates = 27,
+    PreCommitSectorBatch2 = 28,
+    ProveReplicaUpdates2 = 29,
```

Comment:
This is different than the [plan in the FIP](https://github.com/filecoin-project/FIPs/blob/master/FIPS/fip-0034.md#backwards-compatibility).  After thinking about it I don't necessarily disagree with the way you're going.  Where can I read about your thought process to get here?